test-suite/src/lib.rs

```diff
@@ -19,6 +19,7 @@ pub mod nested_select;
 pub mod nullable;
 pub mod order_by;
 pub mod ordering;
+pub mod showcolumns;
```

Comment:
You also need to specify the test like this below, then gluesql integration test will recognize it.
![image](https://user-images.githubusercontent.com/2025065/163702598-90e9c1e6-91d2-4bad-aef3-5473dc2fbe81.png)

Reply:
Thanks for point it out.  I think I got it resolved.